.circleci/config.yml

```diff
@@ -81,3 +93,32 @@ workflows:
             branches:
               only:
                 - force-benchmarks
+
+  scheduled-parsing-stats:
+    triggers:
+      - schedule:
+          # Run at 00:00 every day, UTC.
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - develop
+                - main
```

Comment:
Why run on main? (I'm not actually clear on what main is in semgrep-core)

Reply:
There's no `main` branch in semgrep-core. In the code above, I mean "any branch that looks like the default branch". `main` is the default branch for new repos initialized on GitHub, and hopefully soon the default for `git init` as well. I'm just expecting that some day we'll switch to using `main` instead of `develop`.